src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import SearchResults from '../components/SearchResults';
+
+describe("SearchResults", () => {
+  
+  const { asFragment } = render(<SearchResults/>);
+
```

Comment:
Again, we could probably add some more testing here. I don't think a snapshot would be enough if you knew that your component would be rendering different things depending on props and state